published/003-2016-Jan-12.md

```diff
@@ -0,0 +1,127 @@
+* [**dist.ipfs.io**](https://dist.ipfs.io) A distributions page has been shipped! This was largely spearheaded by [@dignifiedquire](//github.com/dignifiedquire). http://dist.ipfs.io/
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 0.3.11 is shipped! Also, we merged 0.4.0 into master. [@jbenet](//github.com/jbenet), [@Dignifiedquire](//github.com/Dignifiedquire) and others [fixed appveyor](//github.com/ipfs/go-ipfs/pull/2137), which means that windows CI now works. And finally the latest webui by [@Dignifiedquire](//github.com/Dignifiedquire) was pushed in 0.3.11. Check out the [changelog](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md).
+* [**ipfs-merkle-dag**](//github.com/vijayee/ipfs-merkle-dag) [@diasdavid](//github.com/diasdavid) worked on the js ipfs merkle-dag implementation, as well as putting in a lot of work in general on js-ipfs.
+* [**py-ipfs**](//github.com/ipfs/py-ipfs) Thanks to [@candeira](//github.com/candeira) and [@ivilata](//github.com/ivilata), we now have a much more filled out plan for the Python implementation of IPFS. Join in if you are a pythonista.
+* [**js-mafmt**](//github.com/whyrusleeping/js-mafmt) [@whyrusleeping](//github.com/whyrusleeping) is working on a multiaddr validator in js, after his go work in [mafmt](//github.com/whyrusleeping/mafmt)
+* [**infrastructure**](//github.com/ipfs/infrastructure) The public gateway at [ipfs.io](https://ipfs.io) is now backed by both v0.4.0-dev and v0.3.11 at the same time. Requests are proxied to both, and the first successful response wins. ipfs.io will be backed by v0.4.x alone eventually. What this means is that if you want to target your requests at one specific version, use v04x.ipfs.io or v03x.ipfs.io, respectively. The latter will stop returning DNS records when v03x isn't supported on the public gateway. Also, while we're releasing v0.4.0 and for a while after, the default bootstrap nodes are divided among v03x and v04x. Most can be used by v04x nodes, and a few by v03x nodes. The latter will be phased out over time.
+* [**ifps-firefox-addon**](//github.com/lidel/ipfs-firefox-addon/releases/tag/v1.4.2) The Firefox addon that provides transparent access to IPFS resources via local HTTP2IPFS gateway has been updated to release 1.4.2. This is the first time we've mentioned this in this newsletter; go check it out. It also now supports the IPFS protocol scheme.
```

Comment:
Release page for 1.4.2 does not tell much :) Perhaps it is a better idea for the name "ipfs-firefox-addon" to be a  link to https://github.com/lidel/ipfs-firefox-addon#ipfs-firefox-addon? 

Then the phrase "release 1.4.2" could link to https://github.com/lidel/ipfs-firefox-addon/releases/tag/v1.4.2